native/chat/chat-thread-list.react.js

```diff
@@ -44,9 +43,13 @@ const floatingActions = [
 type Item = ChatThreadItem | {| type: 'search', searchText: string |};
 
 type Props = {|
-  navigation: ChatNavigationProp<'ChatThreadList'>,
+  navigation: ChatTopTabsNavigationProp<
```

Comment:
The code handling `onTabPress` below needs to be changed now. `tabNavigation` used to be `navigation`'s parent, but now it is its grandparent